packages/uni_app/lib/view/common_widgets/pages_layouts/general/widgets/top_navigation_bar.dart

```diff
@@ -4,46 +4,54 @@ import 'package:uni/view/common_widgets/page_title.dart';
 class AppTopNavbar extends StatelessWidget implements PreferredSizeWidget {
   const AppTopNavbar({
     this.title,
-    this.rightButton,
```

Comment:
You removed the possibility of any `AppTopNavbar` to have a right button. On some secondary pages, like the exam's page, some icons need to be present. You need to remove the `ProfileButton` from the primary general page, nothing more

Reply:
Thank you !! Fixed :)